plugins/dev-tools/src/generateFieldTestBlocks.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview A field test helper that generates blocks with the field in
+ * various configurations.
+ * @author samelh@google.com (Sam El-Husseini)
+ */
+
+import * as Blockly from 'blockly/core';
+
+
+/**
+ * Generates a number of field testing blocks for a specific field and returns
+ * the toolbox xml string.
+ * @param {string} fieldName The name of the field.
+ * @param {Object|Array.<Object>=} options An options object containing a label
```

Comment:
Based on [this](https://github.com/google/closure-compiler/wiki/Types-in-the-Closure-Type-System) the dot syntax ".<>" seems to be deprecated syntax

Reply:
Yes, good point we should move away from that syntax in this repo.